lib/codecs/src/decoding/format/gelf.rs

```diff
@@ -64,12 +64,12 @@ impl GelfDeserializerConfig {
             [log_namespace],
         )
         .with_event_field(&owned_value_path!(VERSION), Kind::bytes(), None)
-        .with_event_field(&owned_value_path!(HOST), Kind::bytes(), None)
+        .with_event_field(&owned_value_path!(HOST), Kind::bytes(), Some("host"))
         .with_event_field(&owned_value_path!(SHORT_MESSAGE), Kind::bytes(), None)
         .optional_field(&owned_value_path!(FULL_MESSAGE), Kind::bytes(), None)
-        .optional_field(&owned_value_path!(TIMESTAMP), Kind::timestamp(), None)
+        .optional_field(&owned_value_path!(TIMESTAMP), Kind::timestamp(), Some("timestamp"))
         .optional_field(&owned_value_path!(LEVEL), Kind::integer(), None)
-        .optional_field(&owned_value_path!(FACILITY), Kind::bytes(), None)
+        .optional_field(&owned_value_path!(FACILITY), Kind::bytes(), Some("service"))
```

Comment:
Are you sure about this one? GELF actually lists it as deprecated.

https://go2docs.graylog.org/current/getting_in_log_data/gelf.html